kind.yaml

```diff
@@ -8,7 +8,7 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
-  image: kindest/node:v1.16.15
+  image: kindest/node:v1.19.11
```

Comment:
What does this mean for minimum version of Fission? 
Only one test failed on 1.16, so we could still say we support 1.16?

Reply:
Fission installation already says 1.19+ Kubernetes version. So we won't support any issues raised by users, user can run if they don't have an option but officially we won't have the bandwidth to support it.

![image](https://user-images.githubusercontent.com/1511975/119294824-fbb26c00-bc72-11eb-9e4c-b3adaade0b41.png)